pkg/controller/vitessshard/reconcile_tablets.go

```diff
@@ -360,3 +361,54 @@ func tabletAvailableStatus(resultBuilder *results.Builder, pod *corev1.Pod, read
 	resultBuilder.RequeueAfter(tabletAvailableTime - readyDuration)
 	return corev1.ConditionFalse
 }
+
+func (r *ReconcileVitessShard) checkPVCDiskUpdates(ctx context.Context, tabletSpec *vttablet.Spec, pod *corev1.Pod) {
```

Comment:
It's not obvious to me as a reader, based on this function name, that this functions purpose is either to bail, or mutate the tablet specs annotations. I would prefer a name that clearly communicates mutation is going to happen. Suggestion:
```suggestion
func (r *ReconcileVitessShard) updatePvcFilesystemResizeAnnotation(ctx context.Context, tabletSpec *vttablet.Spec, pod *corev1.Pod) {
```

Reply:
Done, changed it to `UpdatePVCFilesystemResizeAnnotation`.